WcaOnRails/app/webpacker/components/DelegateProbations/index.jsx

```diff
@@ -75,9 +76,18 @@ export default function DelegateProbations() {
   return (
     <>
       <h1>Delegate Probations</h1>
-      <Input value={wcaId} onChange={setWcaId} placeholder="Enter WCA ID" />
+      <UserSearch
+        value={userId}
+        onChange={setUserId}
+        multiple={false}
+        delegateOnly
+      />
       <Button
-        onClick={() => save(startDelegateProbationUrl, { wcaId }, sync, { method: 'POST' })}
+        onClick={() => {
+          save(startDelegateProbationUrl, { userId }, sync, { method: 'POST' });
+          setUserId(null);
```

Comment:
The `setUserId` action should be part of the success action in the `save` invocation (the part that currently forwards to `sync`)

Reply:
Okay, done.